### ISSUE TITLE
Find usages should jump to single usage if an spec/fdef exists

### ISSUE THREAD

**Issue by danielcompton** [cursive-ide/cursive#1871]
If you use Find Usages on a function, if there is only a single usage of that function in scope then IntelliJ jumps to that function. However if you have an `fdef` on that function then that counts as another usage of the function, and you are presented with the list of the two functions.

It would be great to have Cursive ignore the fdef for the purposes of jumping to the single implementation, or perhaps there is another metadata category that it could go into (under the Var category?).

**Comment by cursive-ide** [comment]
Yes, `fdef` does seem like a special case that shouldn't count as a normal usage. I'm not sure what I can do about that, but I'll take a look.
